nilearn/conftest.py

```diff
@@ -380,6 +380,15 @@ def img_atlas(shape_3d_default, affine_mni):
 
 
 # ------------------------ SURFACE ------------------------#
+@pytest.fixture
+def single_mesh(rng):
+    """Create random coordinates and faces for a single mesh.
+
+    This does not generate meaningful surfaces.
```

Comment:
Wouldn't it be better to generate a meanginful surface ?

Reply:
A the moment this does exactly the same as what this function does got other tests.

https://github.com/nilearn/nilearn/blob/c984ebe0b52c6cfc9cf7d5f33979250f05866767/nilearn/surface/tests/_testing.py#L9-L18

more refactoring of the test fixtures is required for the surface part:
see https://github.com/nilearn/nilearn/issues/4686

but i'd prefer to keep that for another PR